### ISSUE TITLE
Detailed results not written to after 60 seconds, exiting!

### ISSUE THREAD

**Issue by Didayolo** [codalab/codabench#748]
In CodaBench, if the `detailed_results.html` file is not created, the compute worker stop the execution (without logs on the platform side by the way).

I guess the idea behind this is to be able to have a look at the detailed results during the execution, in real time.

This is cool but not really robust. This causes error because usually this file is created at the end of the execution (especially in old CodaLab bundles).

See:

https://github.com/codalab/codabench/blob/232490ddf2682b89feedc2f6b907e88110828077/docker/compute_worker/compute_worker.py#L242

Update: if the `detailed_results.html` file is empty, it creates a bug.

**I think the file should be created by Codabench (the compute worker) itself if it does not exist.**

**Comment by Didayolo** [comment]
Update: if the `detailed_results.html` file is empty, it creates a bug.

**Comment by Didayolo** [comment]
The file should be created by CodaLab itself if it does not exist.

**Comment by Didayolo** [comment]
I have this error again on another challenge:

```
Detailed results not written to after 60 seconds, exiting!
```

I think it is important that we fix it, because when the compute worker exit like this, the submission fails with no logs at all.

**Comment by Didayolo** [comment]
Another take could be simply to remove this expiration of the scoring program.

**Comment by LidiaGarrucho** [comment]
Hello,
Running the competition in a compute worker I still get:
Detailed results not written to after 60 seconds, exiting!
I have used the latest: codalab/competitions-v2-compute-worker:gpu1.1 in the compute worker.

**Comment by LidiaGarrucho** [comment]
Using docker pull codalab/competitions-v2-compute-worker:gpu1.1 in the compute worker, I am getting:
Detailed results not written to after 60 seconds, exiting!

Why is this? Thank you!

**Comment by LidiaGarrucho** [comment]
No, I am informing this error didn't dissapear when writing Detailed results running a competition on a compute worker. From this thread seems the scoring timeout was deleted but I am getting this message on the console when writing detailed results, even though at the Detailed results are submitted and displayed correctly in HTML format.

**Comment by Didayolo** [comment]
> No, I am informing this error didn't dissapear when writing Detailed results running a competition on a compute worker. From this thread seems the scoring timeout was deleted but I am getting this message on the console when writing detailed results, even though the Detailed results are submitted and displayed correctly in HTML format.

It is actually a warning and not an error. I'll update the message so it is clearer (removing the "exiting" especially). Thank you for the report.
